Will Heikoff Documentation

```diff
@@ -0,0 +1,13 @@
+Found zip code file for line 22: ZIPS.shp
```

Comment:
Please add the full path that points to where you found this file. If you can summarize in a sentence or two HOW you know that this is the right file, that would be ideal.